docs/rules/no-unused-expressions.md

```diff
@@ -34,6 +41,34 @@ f(0), {}
 a && b()
 ```
 
+If you enable the `allowShortCircuit` and/or `allowTernary` options when required you can use expressions like the following but keep in mind that the same restriction on expression side effects still applies:
+
+```js
+a && b()
+```
```

Comment:
I did them as separate code blocks for consistency because I saw that the other examples in this doc were separated as one block per example. When I combine these into one code block would you like me to do the same with the other examples (again for the sake of consistency)?